spring-core-5/src/main/java/com/baeldung/springbean/naming/service/MessagingServiceImpl.java

```diff
@@ -0,0 +1,14 @@
+package com.baeldung.springbean.naming.service;
+
+import com.baeldung.springbean.naming.component.MyCustomComponent;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+
+@Service
+public class MessagingServiceImpl implements MessagingService{
```

Comment:
minor formatting issue here with the opening parenthesis after implements

Reply:
fixed